### ISSUE TITLE
Discard Button

### ISSUE THREAD

**Issue by JPluess** [opnsense/core#4668]
Important notices

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I have searched the existing issues and I am convinced that mine is new.

**Is your feature request related to a problem? Please describe.**

Version 21.1
I would like a discard button if I don’t like the changes. 
Why ask to save changes if no option to discard exists? Does not make too much sense.

Missing Discard Button:
![iphjbkifkglbgknl](https://user-images.githubusercontent.com/7995240/106780388-394da380-6648-11eb-995d-d2bca208c9ee.png)

**Describe the solution you like**

With Discard Button:
![iphjbkifkglbgknl2](https://user-images.githubusercontent.com/7995240/106781421-559e1000-6649-11eb-8327-ed4d1c8d89e6.png)

**Describe alternatives you considered**
Logging out of the System. After login old changes are still asked to be approved.

**Comment by fichtner** [comment]
You can restore any known backup / configuration change via System: Configuration: Backups.


Cheers,
Franco

**Comment by JPluess** [comment]
hi there.  thanks :)
That would only partially work, considering I even backed up the recent previous changes.  

Today I logged in and was prompted with the apply changes banner, because I made some changes a few days back but did not safe the last change. I did not remember exactly what I changed that day. 

A discard button would solve the issue, and no need to run a backup for every little change.
